icons/BlockIcon.tsx

```diff
@@ -2,13 +2,13 @@ import { FC } from "react"
 import { Icon, IconProps } from "@ironfish/ui-kit";
 
 const BlockIcon: FC<IconProps> = (props) => (
-  <Icon viewBox="0 0 26 30" {...props}>
+  <Icon viewBox="0 0 26 30" color="#2C72FF" {...props}>
         <g clipPath="url(#a)">
         <path d="M3.345 22.745h-.053l.128.073-.048-.028-.027-.045ZM3.35 22.745l.048.027.04.023-.088-.05Z"
-            fill="#2C72FF" />
+            fill="currentColor" />
```

Comment:
Interesting, I didn't know this worked.

Reply:
Yes, this is a feature of SVG that is similar to `color: inherit` in `css`.